### PR TITLE
completed task 6 with implemented /api/articles/:article_id/comments …

### PR DESCRIPTION
**Pull request by gilbertouk**
…endpoint

### CODE REVIEW

File: __tests__/api/app.test.js [hunk 1]
```diff
@@ -176,4 +176,70 @@ describe('Testing app', () => {
...
+                author: 'icellusedkars',
+                body: 'git push origin master',
+                article_id: 3,
+              },
+            ];
+
+            expect(comments).toHaveLength(2);
+            expect(comments).toMatchObject(expectComments);
+          });
+      });
```
Comment (B7Bprog) [inline __tests__/api/app.test.js:213]: Please add another test (200) where the article has no comments. This should respond with an empty array.

File: controllers/comments.controllers.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { selectCommentsByArticleId } = require('../models/comments.models');
+
+const getAllCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
```
Comment (B7Bprog) [inline controllers/comments.controllers.js:5]: You will need a way to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.

For this think about using Promise.all()

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hi Gilberto, nice work so far! Please see my comments, refactor your code, then send us another PR with it. 👍

File: controllers/comments.controllers.js (hunk 2 above)
Comment (gilbertouk) [reply controllers/comments.controllers.js:5]: Done

File: __tests__/api/app.test.js (hunk 1 above)
Comment (gilbertouk) [reply __tests__/api/app.test.js:213]: Done

**Review by B7Bprog — APPROVED** [review]
Hey Gilberto, nice refactoring, please merge! 🎉

**Pull request merged**